src/fetch-images.ts

```diff
@@ -8,10 +8,18 @@ import { getFile } from './get-file'
 import { incrementIds } from './utils'
 
 const MAX_CHUNK_SIZE = 1000
+const MAX_RETRIES = 3
+const IMAGE_FETCH_TIMEOUT = 10000
 
-function getImageFromSource(url) {
+function getImageFromSource(
+  page,
+  url,
```

Comment:
Can we flip these parameters? This way it won't be a breaking change.

Reply:
Done